File: BackEnd/src/main/java/com/unosquare/admin_core/back_end/dto/EmployeeSnapshotDto.java
package com.unosquare.admin_core.back_end.dto;

import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor
public class EmployeeSnapshotDto {
    private final int teamId;
    private final String name;
    private final String description;
    private String teamName;
    private int employeeId;
    private String email;
    private String clientName;
}

File: v2/AdminCore.DTOs/Dashboard/EmployeeSnapshotDto.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace AdminCore.DTOs.Dashboard
{
    public class EmployeeSnapshotDto
    {
      public int TeamId { get; set; }

      public string Name { get; set; }

      public string Description { get; set; }

      public string TeamName { get; set; }

      public int EmployeeId { get; set; }

      public string Email { get; set; }

      public string ClientName { get; set; }
  }
}
